linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_keybinding/ansible/shared.yml

```
@@ -3,4 +3,4 @@
 # strategy = configure
 # complexity = low
 # disruption = low
-{{{ ansible_only_lineinfile(None, "/etc/tmux.conf", "\s*bind\s+\w\s+lock-session.*$", "bind X lock-session", create=create, mode="0644") }}}
+{{{ ansible_only_lineinfile(None, "/etc/tmux.conf", "\s*bind\s+\w\s+lock-session.*$", "bind X lock-session", create="yes", mode="0644") }}}
```

```suggestion
{{{ ansible_only_lineinfile(None, "/etc/tmux.conf", "\s*bind\s+\w\s+lock-session.*$", "bind X lock-session", create="true", mode="0644") }}}
```
Per the style guide (https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible)